test/csuite/incr_backup/main.c

```diff
@@ -547,7 +547,7 @@ reopen_file(int *fdp, char *buf, size_t buflen, const char *filename, int oflag)
  */
 static void
 incr_backup(WT_CONNECTION *conn, const char *home, const char *backup_home, TABLE_INFO *tinfo,
-  ACTIVE_FILES *master_active)
+  ACTIVE_FILES *primary_active)
 {
```

Comment:
Although you haven't asked for a review from anyone yet, I just wanted to note that in this context the term does not refer to a replication group. I think a more accurate substitution would be `current_active`.

Reply:
Thanks @sueloverso, I was planning to ask your thoughts and made the requested change.